### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by Andy-Python-Programmer**
Some Minor Fixes:

1. Adding space to the `> `  indicator makes the REPL look nicer and also easy to read and debug ;)

### CODE REVIEW

**Review by Calamity210 — CHANGES_REQUESTED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -3,4 +3,43 @@ project(Senegal C)
...
+    src/sconstant_utils.c 
+    src/includes/stable_utils.h 
+    src/stable_utils.c 
+    src/includes/sopcodes.h 
+    src/core/sboolCore.h 
+    src/core/sboolCore.c 
+    src/core/sstringCore.h 
+    src/core/sstringCore.c 
+    src/core/snumCore.h 
+    src/core/snumCore.c
```
[the remark this answers is not in the document]
Comment (Andy-Python-Programmer) [reply CMakeLists.txt:44]: Oh

File: src/main.c [hunk 2]
```diff
@@ -7,8 +7,9 @@
 
 static void repl(VM* vm) {
   char line[1024];
-  for (;;) {
-    printf(">");
+  
+  while (true) {
```
[the remark this answers is not in the document]
Comment (Andy-Python-Programmer) [reply src/main.c:11]: The was no difference while compiling but ;; just means infinity ♾ loop where while true one is better to read. Maybe it’s just my opinion :) So should I revert that? @Calamity210

[the remark this answers is not in the document]
Comment (Calamity210) [reply src/main.c:11]: Yes please

**Review by Calamity210 — APPROVED** [review]

**Comment by Calamity210** [comment]
Holding on merging for now

**Pull request merged**